quicksettings/QuickSettings.hx

```diff
@@ -331,38 +359,38 @@ extern class QuickSettings
 typedef QSettings =
 {
 	/** optional string, default "QuickSettings" **/
-	@optional var title : String;
+	@:optional var title : String;
```

Comment:
this is correct, the previous version was an oepsie
(https://haxe.org/manual/type-system-typedef.html#optional-fields)